llvm/test/Assembler/ConstantExprFoldCast.ll

```diff
@@ -5,8 +5,8 @@
 
 ; CHECK-NOT: bitcast
 ; CHECK-NOT: trunc
-; CHECK: addrspacecast
-; CHECK: addrspacecast
+; CHECK: ptr addrspace(1) null
+; CHECK: ptr null
```

Comment:
Unrelated to this PR but these checks are not great, can we regen this test with UTC and --check-globals and pre-commit that?